backends/dpdk/spec.cpp

```diff
@@ -343,8 +343,24 @@ std::ostream &IR::DpdkTable::toSpec(std::ostream &out) const {
         0) {
         out << " args none ";
     } else {
-        BUG("non-zero default action arguments not supported yet");
+        out << " args ";
+        auto mce = default_action->to<IR::MethodCallExpression>();
+        auto earg = mce->arguments->at(0)->expression;
+        if (earg->is<IR::ListExpression>()) {
+            auto paramCount = earg->to<IR::ListExpression>()->components.size();
+            for (unsigned i = 0; i < paramCount; i++) {
+                if (earg->to<IR::ListExpression>()->components.at(i)->is<IR::Constant>()) {
```

Comment:
so if the argument is not a numeric constant what should happen?

Reply:
Addressed